{
  "errors": [
    {
      "test": "unittest.loader._FailedTest.OBD_tests",
      "traceback": "ImportError: Failed to import test module: OBD_tests\nTraceback (most recent call last):\n  File \"/usr/lib/python3.6/unittest/loader.py\", line 153, in loadTestsFromName\n    module = __import__(module_name)\n  File \"/home/not-yeshwanth-reddy/Documents/Sem6/Capstone/OBD/src/unittest/python/OBD_tests.py\", line 4, in <module>\n    import OBD\n  File \"/home/not-yeshwanth-reddy/Documents/Sem6/Capstone/OBD/src/main/python/OBD.py\", line 25, in <module>\n    import cv2\nModuleNotFoundError: No module named 'cv2'\n\n"
    }
  ],
  "failures": [],
  "tests-run": 1
}